purchase_order_fixed_price/models/purchase_order.py

```diff
@@ -0,0 +1,15 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields, api
+
```

Comment:
Between imports and class definition should be placed two empty lines, not only one